### PR TITLE
Stream out docker-build cmd stdout

### PR DESCRIPTION
**Pull request by zionrubin**
### *Description of changes:*

For more information on contributing, please see [Contributing](../CONTRIBUTING.md)
section of our documentation.

### Status

- DONE 

### Checklist (tick everything that applies)

- [ ] [PreCommitChecks - Code linting](../CONTRIBUTING.md#InstallRunPreCommit) (required)
- [ ] [Tests](../CONTRIBUTING.md#RunningTests)

### CODE REVIEW

File: liminal/build/image_builder.py [hunk 1]
```diff
@@ -69,25 +69,28 @@ def build(self):
...
-        except subprocess.CalledProcessError as e:
-            docker_build_out = e.output
-            raise e
-        finally:
+            build_process = subprocess.Popen(docker_build_command, shell=True, stdout=subprocess.PIPE)
+            # Poll process.stdout to show stdout live
             logging.info('=' * 80)
-            for line in str(docker_build_out)[2:-3].split('\\n'):
-                logging.info(line)
+            while True:
```
Comment (zionrubin) [inline liminal/build/image_builder.py:76]:
```suggestion
            build_start = time.time()
            timeout = 960
            while time.time() < build_start + timeout:
```

**Comment by liorsav** [comment]
Lgtm

**Comment by liorsav** [comment]
Unit test fsiling please fix

**Comment by zionrubin** [comment]
> Unit test fsiling please fix

i think im ready

**Comment by zionrubin** [comment]
@liorsav i think im ready

**Review by aviemzur — APPROVED** [review]

**Pull request merged**